solidity/package.json

```diff
@@ -5,10 +5,12 @@
   "dependencies": {
     "@eth-optimism/contracts": "^0.6.0",
     "@hyperlane-xyz/utils": "3.5.1",
+    "@layerzerolabs/lz-evm-oapp-v2": "2.0.2",
     "@openzeppelin/contracts": "^4.9.3",
     "@openzeppelin/contracts-upgradeable": "^v4.9.3"
   },
   "devDependencies": {
+    "@layerzerolabs/solidity-examples": "^1.1.0",
```

Comment:
I think this has to be a dependency if we are importing from it in our published contracts?

Reply:
oh nvm
you imported it in LayerZeroV1Hook.sol but its unused

Reply:
yup, removed the unused import